pom.xml

```diff
@@ -87,30 +77,10 @@
             <artifactId>unirest-java</artifactId>
             <version>1.4.9</version>
         </dependency>
-        <dependency>
-            <groupId>org.apache.httpcomponents</groupId>
-            <artifactId>httpclient</artifactId>
-            <version>4.5.2</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.httpcomponents</groupId>
-            <artifactId>httpasyncclient</artifactId>
-            <version>4.0.2</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.httpcomponents</groupId>
-            <artifactId>httpmime</artifactId>
-            <version>4.3.6</version>
-        </dependency>
-        <dependency>
-            <groupId>org.json</groupId>
-            <artifactId>json</artifactId>
-            <version>20140107</version>
-        </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.7.4</version>
+            <version>2.9.8</version>
```

Comment:
Just to be sure, did you test this change? I wouldn't want it to break anything.

Reply:
I expect [JSONUtilTest](https://github.com/jenkinsci/statistics-gatherer-plugin/blob/statistics-gatherer-1.1.3/src/test/java/org/jenkins/plugins/statistics/gatherer/util/JSONUtilTest.java) to accomplish this. I had a look and it covers a least all methods of `JSONUtil`.

Reply:
The old 2.7.4 was released on 29-Apr-2016 (see https://github.com/FasterXML/jackson/wiki/Jackson-Releases). In the meantime a lot of security patches have been applied. Therefore I strongly suggest to upgrade this dependency.